### ISSUE TITLE
Add password

### ISSUE THREAD

**Issue by yas9999** [asapach/peerflix-server#77]
Hello,
i wonder if there is any possibility to protect my server with user & pass just in example.com:9000
& not download link.

best regards

**Comment by Porco-Rosso** [comment]
You should be able to do this by using a reverse proxy. Nginx is commonly used.

**Comment by asapach** [comment]
Yeah, people have been asking this a couple of times: #18, #32 and #54

**Comment by bulltech** [comment]
why not addability to upload torrents directly to google drive

**Comment by maxerbox** [comment]
https://github.com/asapach/peerflix-server/wiki/How-to-put-a-password-on-peerflix-server

**Comment by asapach** [comment]
@maxerbox, thanks, I'll put it into FAQ.
Squid is not ideal, though, because it doesn't proxy websockets. If somebody could come up with a similar guide for nginx or other reverse proxies that would be great.

**Comment by maxerbox** [comment]
@asach You're welcome ! Looks like it's working without websockets. The frames are converted to http request : https://i.imgur.com/teF9g4l.png

I'm too busy to create a nginx tutorial : I'm currently watching a movie with peerflix server :P

**Comment by asapach** [comment]
Yeah, socket.io falls back to long-polling, etc. so it's working, but as I said it's not ideal.
Thanks again.
